<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDebug</class>
 <widget class="QMainWindow" name="MainDebug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>111</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>串口设置</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>95</width>
       <height>356</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>串口</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>ttyS0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ttyS1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ttyS2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ttyUSB0</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>波特率</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>56000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>76800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>128000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>256000</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>数据位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_3">
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>校验位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_4">
        <item>
         <property name="text">
          <string>无</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>奇</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>偶</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>停止位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_5">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>检测串口</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>连续串口</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>间隔（毫秒）</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_6">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>200</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>接受数据</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>330</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>发送数据</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>460</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>字符-字符</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>字符-HEX</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="text">
         <string>DEC_字符</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber_2">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(170, 255, 127)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clear_btn">
        <property name="text">
         <string>清空显示</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>461</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLCDNumber" name="lcdNumber">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(170, 255, 127)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>471</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_C">
    <property name="title">
     <string>操作（C）</string>
    </property>
    <addaction name="action"/>
    <addaction name="separator"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="separator"/>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
    <addaction name="separator"/>
    <addaction name="action_10"/>
    <addaction name="separator"/>
    <addaction name="action_12"/>
    <addaction name="action_13"/>
    <addaction name="separator"/>
    <addaction name="action_15"/>
   </widget>
   <widget class="QMenu" name="menu_A">
    <property name="title">
     <string>关于（A）</string>
    </property>
    <addaction name="action_Lincom"/>
   </widget>
   <addaction name="menu_C"/>
   <addaction name="menu_A"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="icon">
    <iconset resource="image/res.qrc">
     <normaloff>:/add.png</normaloff>:/add.png</iconset>
   </property>
   <property name="text">
    <string>添加串口</string>
   </property>
  </action>
  <action name="action_3">
   <property name="icon">
    <iconset resource="image/res.qrc">
     <normaloff>:/open.png</normaloff>:/open.png</iconset>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset resource="image/res.qrc">
     <normaloff>:/close.png</normaloff>:/close.png</iconset>
   </property>
   <property name="text">
    <string>关闭串口</string>
   </property>
  </action>
  <action name="action_6">
   <property name="icon">
    <iconset resource="image/res.qrc">
     <normaloff>:/keep.png</normaloff>:/keep.png</iconset>
   </property>
   <property name="text">
    <string>保存数据</string>
   </property>
  </action>
  <action name="action_7">
   <property name="icon">
    <iconset resource="image/res.qrc">
     <normaloff>:/read.png</normaloff>:/read.png</iconset>
   </property>
   <property name="text">
    <string>读取文件</string>
   </property>
  </action>
  <action name="action_8">
   <property name="icon">
    <iconset resource="image/res.qrc">
     <normaloff>:/write.png</normaloff>:/write.png</iconset>
   </property>
   <property name="text">
    <string>写入文件</string>
   </property>
  </action>
  <action name="action_10">
   <property name="icon">
    <iconset resource="image/res.qrc">
     <normaloff>:/time.png</normaloff>:/time.png</iconset>
   </property>
   <property name="text">
    <string>读取间隔</string>
   </property>
  </action>
  <action name="action_12">
   <property name="icon">
    <iconset resource="image/res.qrc">
     <normaloff>:/num.png</normaloff>:/num.png</iconset>
   </property>
   <property name="text">
    <string>计数清零</string>
   </property>
  </action>
  <action name="action_13">
   <property name="icon">
    <iconset resource="image/res.qrc">
     <normaloff>:/clear.png</normaloff>:/clear.png</iconset>
   </property>
   <property name="text">
    <string>清空串口</string>
   </property>
  </action>
  <action name="action_15">
   <property name="icon">
    <iconset resource="image/res.qrc">
     <normaloff>:/close_1.png</normaloff>:/close_1.png</iconset>
   </property>
   <property name="text">
    <string>退出程序</string>
   </property>
  </action>
  <action name="action_Lincom">
   <property name="icon">
    <iconset resource="image/res.qrc">
     <normaloff>:/about.png</normaloff>:/about.png</iconset>
   </property>
   <property name="text">
    <string>关于Lincom</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="image/res.qrc"/>
 </resources>
 <connections/>
</ui>
